blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,590 @@
+.v2-footer-wrapper {
+    background: var(--c-primary-black);
+  }
+  
+  .v2-footer {
+    color: var(--c-secondary-silver);
+    padding: 0 0 10px;
+    font-size: 13px;
+    max-width: 100%;
+  }
+
+  .v2-footer .v2-footer-content{
+    padding: 60px 32px 10px;
+    margin: auto;
+    max-width: 1376px;
+  }
+  
+  .v2-footer ul {
+    list-style: none;
+    padding: 0;
+  }
+  
+  .v2-footer .v2-footer-social-media-section {
+    margin-top: 21px;
+    display: flex;
+    gap: 20px;
+    flex-wrap: wrap;
+  }
+  
+  .v2-footer .v2-footer-social-media {
+    border-radius: 17.5px;
```

Comment:
Not sure why `border-radius`, `display`, `font-size`, `text-decoration` is necessary. 🤔 
I would say that colour, width and height is only necessary.

Reply:
Fixed